otter.ts

```diff
@@ -25,10 +24,24 @@ export interface SpeechViewSummary {
     displayId: string
 }
 
+export function getExportTemplate(): string {
+    var exportTemplateConfigVar = process.env.exportTemplate?.trim()
+    if (!exportTemplateConfigVar) {
+        return [
+            "${speech.transcripts.map(it => it.transcript)}",
```

Comment:
feels weird to have this in a template. maybe do something like `getFullTranscript(speech)` or just provide `fullTranscript` var ?